Formula/geckodriver.rb

```diff
@@ -15,10 +15,12 @@ class Wires < Formula
 
   def install
     system "cargo", "build"
-    bin.install "target/debug/wires"
+    bin.install "target/debug/geckodriver"
+    bin.install_symlink bin/"geckodriver" => "wires"
   end
 
   test do
     system "#{bin}/wires", "--help"
+    system "#{bin}/geckodriver", "--help"
```

Comment:
These are effectively the same file (you're symlinking one to the other in the `install` method). Thus it seems unnecessary to test both. I'd suggest dropping the old test in favor of the new one.